crates/ironrdp-client/src/rdp.rs

```diff
@@ -51,26 +58,64 @@ impl RdpInputEvent {
     }
 }
 
+pub struct DvcPipeProxyFactory {
+    rdp_input_sender: mpsc::UnboundedSender<RdpInputEvent>,
```

Comment:
note: It’s typically not ideal to use unbounded channels because there is no backpressure. You may consider a bounded channel and using async `send` or `blocking_send` instead. The sender will have to wait if the queue is full, creating (typically) necessary backpressure when the target system is overloaded.

Reply:
I am re-using the already existing `RdpInputEvent` channel, which is `tokio::mpsc::Unbounded*`, I agree that we need to change that in follow-up